src/BenchmarkDotNet.Diagnostics.Windows/Tracing/NativeMemoryLogParser.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using Microsoft.Diagnostics.Tracing.Etlx;
+using Microsoft.Diagnostics.Tracing.Parsers.Kernel;
+using Address = System.UInt64;
+
+namespace BenchmarkDotNet.Diagnostics.Windows.Tracing
+{
+    public class NativeMemoryLogParser
```

Comment:
thanks for moving it to a separate type! IMHO the separation of responsibilities is much better now 👍